import_export/management/commands/export.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+
+from import_export.command_utils import (
+    get_default_format_names,
+    get_format_class,
+    get_resource_class,
+)
+
+
+class Command(BaseCommand):
```

Comment:
one thing I noticed when testing this

Apparently it's ok to export binary data to the terminal, but it can mess up the [terminal output](https://askubuntu.com/a/1105350)